bigtable/tests/unit/test_instance.py

```diff
@@ -433,27 +433,109 @@ def _create_instance_request(self, clusters):
         )
 
     def test_update(self):
+        import datetime
+        from google.api_core import operation
+        from google.longrunning import operations_pb2
+        from google.protobuf.any_pb2 import Any
+        from google.cloud.bigtable_admin_v2.proto import (
+            bigtable_instance_admin_pb2 as messages_v2_pb2)
+        from google.cloud._helpers import _datetime_to_pb_timestamp
+        from google.cloud.bigtable import enums
         from google.cloud.bigtable_admin_v2.gapic import (
             bigtable_instance_admin_client)
+        from google.protobuf import field_mask_pb2
+        from google.cloud.bigtable_admin_v2.types import instance_pb2
+        from google.cloud.bigtable_admin_v2.proto import (
+            bigtable_instance_admin_pb2 as instance_v2_pb2)
 
-        api = bigtable_instance_admin_client.BigtableInstanceAdminClient(
-            mock.Mock())
+        NOW = datetime.datetime.utcnow()
+        NOW_PB = _datetime_to_pb_timestamp(NOW)
         credentials = _make_credentials()
         client = self._make_client(project=self.PROJECT,
                                    credentials=credentials, admin=True)
         instance = self._make_one(self.INSTANCE_ID, client,
-                                  display_name=self.DISPLAY_NAME)
+                                  display_name=self.DISPLAY_NAME,
+                                  instance_type=enums.InstanceType.DEVELOPMENT,
+                                  labels=self.LABELS)
+
+        expected_request_instance = instance_pb2.Instance(
+            name=instance.name, display_name=instance.display_name,
+            type=instance.type_, labels=instance.labels)
+        expected_request_update_mask = field_mask_pb2.FieldMask(
+            paths=['display_name', 'type', 'labels'])
+        expected_request = instance_v2_pb2.PartialUpdateInstanceRequest(
+            instance=expected_request_instance,
+            update_mask=expected_request_update_mask)
+
+        metadata = messages_v2_pb2.UpdateInstanceMetadata(
+            request_time=NOW_PB)
+        type_url = 'type.googleapis.com/%s' % (
+            messages_v2_pb2.UpdateInstanceMetadata.DESCRIPTOR.full_name,)
```

Comment:
Please remove the extra `,`

Reply:
Actually, at this point we prefer to use `{}` + `format()` over the `%` operator in new code (and update opportunistically in old code).  E.g.:

```python
        type_url = 'type.googleapis.com/{}'.format(
            messages_v2_pb2.UpdateInstanceMetadata.DESCRIPTOR.full_name)
```

Reply:
Updated to `{}` + `string.format()`